src/Components/ContentProperty.tsx

```diff
@@ -1,9 +1,20 @@
 import * as React from "react";
 import * as Scrivito from "scrivito";
 
+interface ValidationResultObject {
+  severity?: string | undefined;
```

Comment:
`severity?:` equal so `severity: undefined`

no need for undefined in `severity?: string | undefined;`

Reply:
Removed it.